test/e2e/reconciler/cluster/controller_test.go

```diff
@@ -69,7 +69,7 @@ func TestClusterController(t *testing.T) {
 					ObjectMeta: metav1.ObjectMeta{
 						Name: "timothy",
 						Labels: map[string]string{
-							nscontroller.ClusterLabel: sinkClusterName,
```

Comment:
this should rather be a string literal. We want that tests break when we change a label or annotation in the main code.

Reply:
done